docs/output-formats-filters.md

```diff
@@ -0,0 +1,125 @@
+* Output fields may be used to select and order the data columns.
+* Output filters may be used to limit which data rows are printed based on logical expressions.
+
+Output Formats
+==============
+
+The `--format` option may be used to select the data format used to print the output of a command. Most commands that produce informative output about some object or system can transform their data into different formats. For example, the Drush `version` command may be printed in a human-readable table (the default), or in a json array:
+```
+$ drush9 version
+ Drush version : 9.5.0
+$ drush9 version --format=json
+{
+    "drush-version": "9.5.0"
+}
+```
+The available output formats are shown in the `help` for each command:
+```
+$ drush help version
+Show drush version.
+
+Options:
+ --format=<json>    Select output format. Available: json, string, var_export, yaml. Default is key-value.
+```
+
+Output Fields
+=============
+
+If you wish to limit the number of columns produced by a command, use the `--fields` option. List the field names in the order they should be displayed:
+```
+$ drush9 views:list --fields=machine-name,status
++-------------------+----------+
+| Machine name      | Status   |
++-------------------+----------+
+| block_content     | Enabled  |
+| comment           | Enabled  |
+| comments_recent   | Enabled  |
+| content           | Enabled  |
+| content_recent    | Enabled  |
+| files             | Enabled  |
+| frontpage         | Enabled  |
+| taxonomy_term     | Enabled  |
+| user_admin_people | Enabled  |
+| watchdog          | Enabled  |
+| who_s_new         | Enabled  |
+| who_s_online      | Enabled  |
+| archive           | Disabled |
+| glossary          | Disabled |
++-------------------+----------+
+```
+The available field names are shown in the `help` text:
+```
+$ drush9 help views:list
+Get a list of all views in the system.
+
+Options:
+  --fields=FIELDS   Available fields: Machine name (machine-name),     
+                    Name (label), Description (description), Status    
+                    (status), Tag (tag) [default:                      
+                    "machine-name,label,description,status"]           
+```
+Fields may be named either using their human-readable name, or via their machine name.
+
+Note also that some commands do not display all of their available data columns by default. To show all available fields, use `--fields=*`
+
+There is also a singluar form `--field` available. If this form is used, it will also force the output format to `string`.
+```
+$ drush9 views:list --field=machine-name 
+block_content
+comment
+comments_recent
+content
+content_recent
+files
+frontpage
+taxonomy_term
+user_admin_people
+watchdog
+who_s_new
+who_s_online
+archive
+glossary
+```
+
+Output Filters
```

Comment:
I'm thinking that this topic should be offerred automatically in help for any command that shows the --fields option.

Reply:
Done